interview-questions.md

```diff
@@ -9,57 +9,66 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. The built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
   Your answer:
+the built in method map takes (1) Value (2) index and (3)the arrays itself all of them are requiered.
+  Researched answer: The value at each index of an array which is required the index of the array which is optional and the array of which is optional 
+
+
+
 
-  Researched answer:
 
 
 
 2. What is object destructuring?
 
-  Your answer:
+  Your answer: could not remember what destructuring was 
+
+  Researched answer: creating a shortcut to get the value of the nested object 
 
-  Researched answer:
 
 
 
 3. Why would you use the method super()?
 
-  Your answer:
+  Your answer: help pass attributes from the parent classes
 
-  Researched answer:
+  Researched answer: within in the constructor method we call super() which passes the attributes from the constructor in the parent class
 
 
 
 4. What is a React Fragment? Why would you use one?
 
-  Your answer:
+  Your answer: i dont remember seeing this in class 
```

Comment:
It's funny react fragments used to look like <React.Fragment> </React.Fragment> but the devs realized it's too much. So they made it into <> </>